lib/house.rb

```diff
@@ -0,0 +1,66 @@
+class House
+  attr_reader :address, :rooms
+
+  def initialize(price, address)
+    @price = price
+    @address = address
+    @rooms = []
+
+  end
+
+  def price
+    if @price[0] == "$"
```

Comment:
This is an interesting approach! I like that you're thinking about scaling the program to account for multiple scenarios. Can you think of a way to make this more readable with guard statements?